DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Controllers/NoteController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Notes.WebApi.Services;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi.Controllers
+{
+    public class NoteController : ApiController
```

Comment:
Btw do you remember we have a discussion about CQRS design? What the main points? Can you describe it?